postgres/datadog_checks/postgres/postgres.py

```diff
@@ -902,7 +902,7 @@ def _send_database_instance_metadata(self):
                 "collection_interval": self._config.database_instance_collection_interval,
                 'dbms_version': payload_pg_version(self.version),
                 'integration_version': __version__,
-                "tags": self._non_internal_tags,
+                "tags": [t for t in self._non_internal_tags if not t.startswith('db:')],
```

Comment:
Would it be easy to add a test for that?

Reply:
![image](https://github.com/DataDog/integrations-core/assets/1266346/b42bd32b-432d-4d81-a81e-0672c0fbed92)

😅

Reply:
hahaha sorry @FlorentClarret I missed this comment! Yes, I can add a test for this today. Sorry about that!